lib/linux_admin/dns.rb

```diff
@@ -0,0 +1,41 @@
+module LinuxAdmin
+  class Dns
+    attr_accessor :filename
+    attr_accessor :nameservers
+    attr_accessor :search_order
+
+    def initialize(filename = "/etc/resolv.conf")
+      @filename = filename
+      reload
+    end
+
+    def reload
+      parse_file(File.read(@filename).split("\n"))
+    end
+
+    def save
+      contents = "search #{@search_order.join(" ")}\n"
+      @nameservers.each { |ns| contents += "nameserver #{ns}\n" }
+      File.write(@filename, contents)
+    end
+
+    private
+
+    def parse_file(lines)
```

Comment:
I think the `File.read` belongs in this method and it shouldn't accept any params.